style.css

```diff
@@ -543,6 +543,28 @@ q:after {
 	outline: none;
 }
 
+/* Typography for Japanese Font */
+
+html[lang="ja"] body,
+html[lang="ja"] button,
+html[lang="ja"] input,
+html[lang="ja"] select,
+html[lang="ja"] textarea {
+	font-family: "Hiragino Kaku Gothic Pro", Meiryo, sans-serif;
+	letter-spacing: normal;
+}
+
+html[lang="ja"] h1,
+html[lang="ja"] h2,
+html[lang="ja"] h3,
+html[lang="ja"] h4,
+html[lang="ja"] h5,
+html[lang="ja"] h6 {
+	font-family: "Hiragino Kaku Gothic Pro", Meiryo, sans-serif;
```

Comment:
Like we discussed before there is no need to repeat the same CSS for the headers.